shared/util/kbfs.js

```diff
@@ -24,11 +24,3 @@ export function getTLF (isPublic: boolean, basedir: string): string {
     return `/private/${basedir}`
   }
 }
-
-export function cleanup (folderName: string): string {
-  if (!folderName) {
-    return ''
-  }
-
-  return folderName.replace(/\s/g, '').replace(/\.\./g, '').replace(/\//g, '').replace(/\\/g, '')
```

Comment:
this was here to protect user input from entering things like /keybase/private/chrisnojima,../../../etc/passwd